### ISSUE TITLE
Symmetric KL is exploding

### ISSUE THREAD

**Issue by git-jhyang** [wufan2021/Heterogeneous-Graph-Information-Bottleneck#3]
Hi, I got some problem on applying a HGIB on my model.

After few training steps of SSIB, loss (`mi_gradient + skl * args.beta`) was exploted to `-inf`.

I controlled beta but the loss was exploded anyway

I found that `SKL` term decreased toward `-inf` while `mi_gradient` decreased slowly. (but expected to be increased for maximizing `MI` between `z1` and `z2`)

I tried your code with minor editing (adding tensorboard scalers) and found same issue.

Here is results (~800 epochs)
![capture](https://user-images.githubusercontent.com/40656213/182773784-8d275c0e-60be-4a5e-940b-dfa8a553919d.png)

`beta_skl` : `skl * args.beta` value
`loss` : your loss (from source code, `.compute_loss()` function), `-mi_grad + beta_skl`
`mi`: `mi_estimation` value
`mi_grad`: `mi_gradient` value

Is it correct behavior?

**Comment by cuocuo-li** [comment]
Hi,I've encountered the same issue. Have you found a solution?

**Comment by cuocuo-li** [comment]
Please reply to me，thanks
